demos/risk/InfrastructureRisk/RoadNetwork/exposure_demo_edges.csv

```diff
@@ -0,0 +1,39 @@
+id,lon,lat,taxonomy,number,type,start_node,end_node,supply_or_demand,length,parent_id
```

Comment:
@asthapoudel under the `type` column, some entries are capitalized (eg. "Road") while others aren't (eg. "bridge1", "bridge2"). Would be good to harmonize them.

Reply:
Done